files/zh-cn/learn/javascript/asynchronous/promises/index.md

```diff
@@ -180,9 +181,9 @@ Promise 中有一些具体的术语值得我们弄清楚。
 
 注意，这里的“成功”或“失败”的含义取决于所使用的 API：例如，`fetch()` 认为服务器返回一个错误（如[404 Not Found](/zh-CN/docs/Web/HTTP/Status/404)）时请求成功，但如果网络错误阻止请求被发送，则认为请求失败。
 
-有时我们用 **已敲定（settled）** 这个词来同时表示 **已兑现（fulfilled）** 和 **已拒绝（rejected）** 两种情况。
+有时我们用**已敲定（settled）**这个词来同时表示**已兑现（fulfilled）**和**已拒绝（rejected）**两种情况。
```

Comment:
This would not be correctly rendered:

![image](https://github.com/mdn/translated-content/assets/15844309/ae22fe4d-f7a2-485c-b13a-8f013206cf64)

```suggestion
有时我们用**已敲定**（settled）这个词来同时表示**已兑现**（fulfilled）和**已拒绝**（rejected）两种情况。
```